File: DesafioSistemaTarefas.Application/DTOs/HistoricoTarefaDto.cs
using System.Diagnostics.CodeAnalysis;

namespace DesafioSistemaTarefas.Application.DTOs
{
    [ExcludeFromCodeCoverage]
    public class HistoricoTarefaDto : TarefaDto
    {
        public int IdTarefa { get; set; }
        public DateTime? DataHoraExclusao { get; set; }
        public DateTime? DataHoraConclusao { get; set; }
    }
}

File: DesafioSistemaTarefas.Front/src/app/tarefa/models/HistoricoTarefaDto.ts
import { TarefaDto } from "./tarefaDto";

export class HistoricoTarefaDto extends TarefaDto {

    public HistoricoTarefaDto()
    {
        
    }
    IdTarefa: number = 0;
    DataHoraExclusao: Date = {} as Date;
    DataHoraConclusao: Date = {} as Date;
}